hal/src/stm32f2xx/sleep_hal.cpp

```diff
@@ -55,6 +55,33 @@ static constexpr uint8_t GPIO_IRQn[extiChannelNum] = {
     EXTI15_10_IRQn  //15
 };
 
+uint8_t exti0Bumped = false;
+uint8_t exti1Bumped = false;
+uint8_t exti2Bumped = false;
+uint8_t exti3Bumped = false;
+uint8_t exti4Bumped = false;
+uint8_t exti95Bumped = false;
+uint8_t exti1510Bumped = false;
+
+static uint8_t* const pExtiBump[extiChannelNum] = {
```

Comment:
Or just simplify this and use the fact that this is 0-15 and use a bitmask. There is no need to introduce two levels of indirection.